addon/helpers/ember-sparkles/arc.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+import { arc } from 'd3-shape';
+
+export function emberSparklesArc([ radius ], textLabel=false) {
+  if (textLabel === true){
+    return arc().outerRadius(radius - 40).innerRadius(radius - 40);
```

Comment:
we will want to pass `outerRadius` and `innerRadius` arguments to this helper instead of hardcoding their inputs